d3a_interface/sim_results/simulation_assets_info.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict, List
+
+from d3a_interface.sim_results.results_abc import ResultsBaseClass
+
+
+class SimulationAssetsInfo(ResultsBaseClass):
+    FIELDS = ['number_of_house_type', 'avg_assets_per_house', 'number_of_load_type',
```

Comment:
Fields like `avg_assets_per_house` or `number_of_storage_type` could be acquired solely from the configuration and thus only on the d3a-web side.

Reply:
@Hassan754 Sorry, I can not find this in your d3a-web PR. Can you maybe point me to it?